projects/js-packages/publicize-components/src/social-store/actions/social-notes.ts

```diff
@@ -0,0 +1,33 @@
+import { store as coreStore } from '@wordpress/core-data';
+import { SOCIAL_NOTES_CONFIG_KEY, SOCIAL_NOTES_ENABLED_KEY } from '../constants';
+import { SocialNotesConfig } from '../types';
+
+/**
+ * Sets the Social Notes enabled status.
+ *
+ * @param  isEnabled - The new enabled status.
+ *
+ * @return {Function} A thunk.
+ */
+export function setSocialNotesEnabled( isEnabled: boolean ) {
```

Comment:
Should this be named something like `toggleSocialNotes` to make the name work for both enabling and disabling?